OsmAnd/src/net/osmand/plus/views/layers/PointLocationLayer.java

```diff
@@ -58,6 +73,8 @@
 	private float textScale = 1f;
 	@ColorInt
 	private int color;
+	@ColorInt
+	private final static int OUTDATED_COLOR = 0xff777777;
```

Comment:
There are no predefined color for this purposes, and initial `legaccy` implementation didn't use it.
In case of this value define d some where - could you please share it.
In case of this value have to bee added to the settings - then it have to be implemented separately I think so.